quadratic_equation.c

```diff
@@ -1,5 +1,60 @@
 #include "quadratic_equation.h"
 
+int Solve(double first_par, double second_par, double third_par, double* x1, double* x2) {

```

Comment:
What are the possible cases for the number of solutions of a quadratic equation? 0, 1 and 2 are ofc obvious, but can't be there an infinite number of solutions?)